### PR TITLE
[Feature](bangc-ops):fix pr

### PR DESCRIPTION
**Pull request by tudejiang79**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

## 3. Test Report

If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md).

#### 3.1.1 Accuracy Acceptance Standard

For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](../docs/MLU-OPS-Accuracy-Acceptance-Standard.md).

- [ ] diff1, diff1 <= 3e-3
- [ ] diff2, diff2 <= 3e-3

#### 3.1.2 Operator Scheme checklist

|     No.      |           Details            |      Check Results       |
|----------------|---------------------------|---------------------|
|        1       |          Supported hardware         | MLU270 <br> MLU290 <br>MLU370<br>MLU590|
|        2       |          Job types          |    block <br> U1 <br> U4    |
|        3       |         Layouts            |  NHWC 、NCHW、ARRAY etc    |
|        4       |         Whether multi-dimensions are supported              |                |
|        5       |          Whether element zero is supported             |               |
|        6       |         Data type(half/float)       |         half / float etc           |
|        7      |        Whether there is size limit           |             |

#### 3.1.3 New Feature Test

If you have checked the following items, please tick the relevant box.

- [ ] Data type test
- [ ] Multidimensional tensor test
- [ ] Layout test
- [ ] Different size/integer remainder end segment/alignment misalignment test
- [ ] Zero dimensional tensor test/zero element test
- [ ] stability test
- [ ] Multiple platform test
- [ ] Gen_case module test
- [ ] Nan/INF tests 
- [ ] Bug fix tests
- [ ] For memory leak check details, see[GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md).
- [ ] For code coverage check details, see: [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md).
- [ ] For I/O calculation efficiency check details, see: [MLU-OPS Performance Acceptance Standard](../docs/MLU-OPS-Performance-Acceptance-Standard.md).

#### 3.1.4 Parameter Check

When a new operator is submitted, the test points are given and the test results are stated.

| Test Point         | Acceptance Standard | Test Result (Error Message) |
| -------------- | -------- | -------------------- |
| Whether it conforms to the operator restriction | Normal error |                      |
| Whether illegal parameters are passed  | Normal error |                      |

### 3.2 Performance Test

See [MLU-OPS Performance Acceptance Standard](../docs/MLU-OPS-Performance-Acceptance-Standard.md) for details.

Platform ：MLU270

|Operator|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-----|----|----|----|----|----|------|-----|
|op_name|   |    |     |    |    |    |     |
|op_name|   |    |     |    |    |    |     |

Platform ：MLU290

|Operator|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-----|----|----|----|----|----|------|-----|
|op_name|   |    |     |    |    |    |     |
|op_name|   |    |     |    |    |    |     |

Platform：MLU370

|Operator|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-----|----|----|----|----|----|------|-----|
|op_name|   |    |     |    |    |    |     |
|op_name|   |    |     |    |    |    |     |

Platform：MLU590

|Operator|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-----|----|----|----|----|----|------|-----|
|op_name|   |    |     |    |    |    |     |
|op_name|   |    |     |    |    |    |     |

### 3.3 Summary Analysis

Please give a brief overview here, if you want to note and summarize the content.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -60,7 +60,20 @@ When a new operator is submitted, the test points are given and the test results
 | Whether it conforms to the operator restriction | Normal error |                      |
 | Whether illegal parameters are passed  | Normal error |                      |
 
-### 3.2 Performance Test
+### 3.2 Accuracy Test
+For the cases used in the New Feature Test section, the features of these cases are mainly described here.
+
+|Test Point           | Description      | Quantity |  comment |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:66]: Comment

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -60,7 +60,20 @@ When a new operator is submitted, the test points are given and the test results
...
+|----------           |----------        |----------|   ----------    |
+|Data type test       |eg:half/float/int8|          |                 |
+|Mult-tensor test     |eg:support 1-8 dims|          |                 |
+|Layout test          |eg:support NCHW/NHWC|          |                 |
+|zero element test    | support or not     |          |                 |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md]: Zero

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -60,7 +60,20 @@ When a new operator is submitted, the test points are given and the test results
...
+|stability test       |--gtest_repeat=NUM<br>--thread=NUM |          |                 |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md]: Stability

Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 2 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 1 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 3 above)
Comment (GraceGuanH) [inline .github/pull_request_template.md]: Stability

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -60,7 +60,20 @@ When a new operator is submitted, the test points are given and the test results
...
+|Mult-platform test   |eg:270/290/370/590  |          |                 |
+|Nan/INF tests        |   support or not   |          |                 |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md]: tests > test

File: .github/pull_request_template.md (hunk 3 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -60,7 +60,20 @@ When a new operator is submitted, the test points are given and the test results
 | Whether it conforms to the operator restriction | Normal error |                      |
 | Whether illegal parameters are passed  | Normal error |                      |
 
-### 3.2 Performance Test
+### 3.2 Accuracy Test
+For the cases used in the New Feature Test section, the features of these cases are mainly described here.
+
+|Test Point           | Description      | Quantity |  comment |
+|----------           |----------        |----------|   ----------    |
+|Data type test       |eg:half/float/int8|          |                 |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md]: 去掉eg：

File: .github/pull_request_template.md (hunk 4 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -60,7 +60,20 @@ When a new operator is submitted, the test points are given and the test results
...
+|Mult-tensor test     |eg:support 1-8 dims|          |                 |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md]: eg:support 1-8 dims 》Supports 1-8 dims

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -60,7 +60,20 @@ When a new operator is submitted, the test points are given and the test results
...
+|Layout test          |eg:support NCHW/NHWC|          |                 |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md]: Supports NCHW/NHWC.

File: .github/pull_request_template.md (hunk 2 above)
Comment (GraceGuanH) [inline .github/pull_request_template.md]: Whether to support or not.

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -60,7 +60,20 @@ When a new operator is submitted, the test points are given and the test results
...
+|zero element test    | support or not     |          |                 |
+|stability test       |--gtest_repeat=NUM<br>--thread=NUM |          |                 |
+|Mult-platform test   |eg:270/290/370/590  |          |                 |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md]: 270/290/370/590

File: .github/pull_request_template.md (hunk 2 above)
Comment (GraceGuanH) [inline .github/pull_request_template.md]: Support or not

File: .github/pull_request_template.md (hunk 6 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 5 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 7 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 4 above)
Comment (GraceGuanH) [inline .github/pull_request_template.md]: Support or not

File: .github/pull_request_template.md (hunk 2 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 8 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 2 above)
Comment (GraceGuanH) [inline .github/pull_request_template.md]: Whether to support the test.

File: .github/pull_request_template.md (hunk 4 above)
Comment (GraceGuanH) [inline .github/pull_request_template.md]: Whether to support this test.

File: .github/pull_request_template.md [hunk 9]
```diff
@@ -60,7 +60,23 @@ When a new operator is submitted, the test points are given and the test results
...
+For the cases used in the New Feature Test section, the features and the quantity of cases are recorded here. When multiple ops are tested, each op is supposed to record one particular table.
+
+Operator:
+
+|Test Point           | Description      | Quantity |  Comment |
+|----------           |----------        |----------|   ----------    |
+|Data type test       |half/float/int8|          |                 |
+|Mult-tensor test     |Supports 1-8 dims|          |                 |
+|Layout test          |Supports NCHW/NHWC|          |                 |
+|Zero element test    |Whether to support the test |          |                 |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:74]: Whether to support this test

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -60,7 +60,23 @@ When a new operator is submitted, the test points are given and the test results
 | Whether it conforms to the operator restriction | Normal error |                      |
 | Whether illegal parameters are passed  | Normal error |                      |
 
-### 3.2 Performance Test
+### 3.2 Accuracy Test
+
+For the cases used in the New Feature Test section, the features and the quantity of cases are recorded here. When multiple ops are tested, each op is supposed to record one particular table.
+
+Operator:
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:67]: Operation:

File: .github/pull_request_template.md (hunk 2 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md [hunk 11]
```diff
@@ -60,7 +60,23 @@ When a new operator is submitted, the test points are given and the test results
 | Whether it conforms to the operator restriction | Normal error |                      |
 | Whether illegal parameters are passed  | Normal error |                      |
 
-### 3.2 Performance Test
+### 3.2 Accuracy Test
+
+For the cases used in the New Feature Test section, the features and the quantity of cases are recorded here. When multiple ops are tested, each op is supposed to record one particular table.
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:65]: For the cases used in the New Feature Test section, the features and the number of cases are recorded here. When multiple operations are tested, multiple tables are needed to include details of these operations.

File: .github/pull_request_template.md (hunk 4 above)
Comment (tudejiang79) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 9 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:74]: 已修改

File: .github/pull_request_template.md (hunk 10 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:67]: 已修改

File: .github/pull_request_template.md (hunk 11 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:65]: 已修改

**Review by GraceGuanH — APPROVED** [review]

**Review by duzekunKTH — APPROVED** [review]

**Review by WangYuan-cn — APPROVED** [review]

**Pull request merged**